src/app/state/actions/index.tsx

```diff
@@ -379,6 +379,19 @@ export const logInUser =
     }
   };
 
+export const upgradeAccount =
+  (params: { verificationDetails: string; otherInformation?: string }) => async (dispatch: Dispatch<Action>) => {
+    dispatch({ type: ACTION_TYPE.ACCOUNT_UPGRADE_SEND_REQUEST });
+    try {
+      const pendingAccountUpgrade = await api.users.upgradeAccount(params);
+      dispatch({ type: ACTION_TYPE.ACCOUNT_UPGRADE_SEND_RESPONSE_SUCCESS, user: pendingAccountUpgrade.data });
+    } catch (e: any) {
```

Comment:
😢
Having said that, I'm not sure what the right thing to do is here. Maybe something like this?
```typescript
    } catch (e) {
      const errorMessage = extractMessage(e as Error);
```
It may be worth spending some time researching this.